src/service/url-replacements-impl.js

```diff
@@ -462,6 +462,12 @@ export class GlobalVariableSource extends VariableSource {
     this.set('BACKGROUND_STATE', () => {
       return viewerForDoc(this.ampdoc).isVisible() ? '0' : '1';
     });
+
+    // Returns the user agent.
+    this.set('USER_AGENT', () => {
```

Comment:
Please group user_agent to the `Device and Browser` category, around [line](https://github.com/ampproject/amphtml/blob/master/src/service/url-replacements-impl.js#L357).